README.md

```diff
@@ -7,20 +7,71 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
+## Setup
+
+### From CDN
+
+The font can be served from a CDN such as [JSDelivr][jsdelivr-link] or [Unpkg][unkpkg-link]. Simply use the `simple-icons-font` npm package and specify a version in the URL like the following:
+
+```html
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/simple-icons-font@v2/font/simple-icons.min.css" type="text/css">
+```
+
+```html
+<link rel="stylesheet" href="https://unpkg.com/simple-icons-font@2/font/simple-icons.min.css" type="text/css">
+```
+
+### NodeJS
+
+The font is also available through our npm package. To install, simply run:
+
+```
+$ npm install simple-icons-font
+```
+
+After installation, the icon and stylesheet font can then be found in `node_modules/simple-icons-font/font`. You can use your favorite bundling tools to include them into your project.
```

Comment:
Was this a mistake in my suggestion? :fearful: 

```suggestion
After installation, the icon font and stylesheet can be found in `node_modules/simple-icons-font/font`. You can use your favorite bundling tool to include them into your project.
```

Reply:
I'm 100% sure _"then"_ is unnecessary and 90% sure it is incorrect (though I'm not native in English either). This may be a bit easier to see if you simplify the sentence:

```diff
After installation, it can be found [...]
After installation, it can then be found [...]
```